packages/fetchai/skills/tac_negotiation/skill.yaml

```
@@ -76,10 +76,16 @@ models:
     class_name: SigningDialogues
   strategy:
     args:
+      classification:
+        piece: classification
+        value: tac.trader
```

```suggestion
        value: tac.participant
```
to keep it consistent?